pkgs/by-name/wa/watchdog/package.nix

```diff
@@ -0,0 +1,24 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  version = "5.16";
+  pname = "watchdog";
+  src = fetchzip {
+    url = "mirror://sourceforge/watchdog/watchdog-${finalAttrs.version}.tar.gz";
+    hash = "sha256-ecXsnPvAhlRB8jiTgK+i1j6m/0idNqmzjSqi6UGCydE=";
+  };
+  makeFlags = [
+    "CONFIG_FILENAME:=${placeholder "out"}/etc/watchdog.conf"
```

Comment:
Does this prevent user from configuring in e.g. `/etc/watchdog.conf`? Have you tested that?

Reply:
I thought it would be nice to provide an example config file (which works as in for many cases). Also, the build breaks when trying to install the config file outside the store, so this killed two birds with one stone.

You can specify an alternate config file with `-c`, which works well.

If merged, my next step will be to create a module (and hopefully some NixOS tests) to make things even easier.